drivers/storage/portworx/util/util.go

```diff
@@ -962,6 +967,31 @@ func IsTelemetryEnabled(spec corev1.StorageClusterSpec) bool {
 		spec.Monitoring.Telemetry.Enabled
 }
 
+// IsCCMGoSupported returns true if px version is higher than 2.12
+func IsCCMGoSupported(pxVersion *version.Version) bool {
+	// TODO: remove after testing
```

Comment:
Is this comment referring to removal of the ` || true` ? Considering we are prepping the PR to merge, let's remove this now.